lib/ansiblelint/utils.py

```diff
@@ -732,3 +739,89 @@ def get_rule_skips_from_line(line):
         noqa_text = line.split('# noqa')[1]
         rule_id_list = noqa_text.split()
     return rule_id_list
+
+
+def is_playbook(filename):
+    try:
+        f = parse_yaml_from_file(filename)
+        if isinstance(f, AnsibleSequence) and hasattr(f[0], 'hosts'):
```

Comment:
I think this would make more sense in the `else:` branch.

Reply:
That's a conservative function, expected to return True only in idea conditions. I am not aware of a way to rewrite that logic in a more concise way (or at least not without affecting its readability). I am aware that I could remove
the last return and use the implicit None, but is kinda like it bit more explicit on that aspect.

Anyway, if you know a better way, propose a change here, I will gladly accept it.